### PR TITLE
feat: add API v2 spec preview

### PR DESCRIPTION
**Pull request by seymourisdead**
Swagger UI for API v2 is available on `/api-reference/next`, the "try it out button" is hidden and there's a disclaimer that it's a work in progress, @danieltprice please review and fix the copy if needed.

https://websitemain62807-addapiv2previewspec.gtsb.io/api-reference/next/

### CODE REVIEW

File: src/pages/api-reference/next.jsx [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+  React.useEffect(() => {
+    fetch('https://dqjnwjfwjj8yz.cloudfront.net/api_spec/release/v2.json')
+      .then((res) => res.json())
+      .then((spec) => {
+        spec.servers = [
+          // {
+          //   url: 'https://console.neon.tech/api/v2',
+          // },
+        ];
+        spec.info.description = `This is preview of the next version of the API. It's only partially implemented for now and only ment to be reviewed, not used.\n\n${spec.info.description}`;
```
Comment (danieltprice) [inline src/pages/api-reference/next.jsx:17]:
```suggestion
        spec.info.description = `This is a preview of the next version of the Neon API. It is only partially implemented and intended for review purposes only.\n\n${spec.info.description}`;
```

**Review by danieltprice — DISMISSED** [review]
Looks good to me!

**Review by stepashka — COMMENTED** [review]
why hide 'try it' button for things that are implemented?

**Comment by seymourisdead** [comment]
> why hide 'try it' button for things that are implemented, @seymourisdead ?
 
 because you can not filter wich endpoints will have the button, so i decided to hide it for all of them

**Comment by stepashka** [comment]
let's bring it back? it's useful for the endpoints that are implemented

**Review by stepashka — COMMENTED** [review]
let's add the link to the new API spec from the Neon API page of the docs
this shouldn't replace the existing link to the API v1, but add the mention of the new API v2 that is under development
let's also bring back the try it out button to the v2 swagger UI

**Comment by stepashka** [comment]
@danieltprice could you do the APi docs update as per comments above?
in the meanwhile @seymourisdead will bring back the 'try it' buttons

**Comment by danieltprice** [comment]
After comparing V1 and V2 copy, I have an additional comment:

For V1, we have this copy at the top:

**API V1**
Neon Cloud API

For V2, we have this copy at the top:

**Neon public API**
This is a preview of the next version of the Neon API. It is only partially implemented and intended for review purposes only.
Neon public API
Contact the developer
Proprietary

Can we make the copy more consistent? For example:

For V1:

**Neon API**

For V2:

**Neon API**

This is a preview of the next version of the Neon API. It is only partially implemented and intended for review purposes only.

If you encounter issues, please contact [Neon Support](mailto:support@neon.tech).

Any need for "Proprietary"? 
Any need for "API V1" if when we have v1 and v2 tags? 
Any need to identify the API as "public"? I think it's implied by the fact that it's published.

**Comment by seymourisdead** [comment]
brought back the try it out button and changed the url to /api-reference/v2

**Review by stepashka — APPROVED** [review]

**Review by danieltprice — APPROVED** [review]
LGTM

**Pull request merged**